swing/src/net/sf/openrocket/gui/rocketfigure/RailButtonShapes.java

```diff
@@ -27,18 +31,16 @@
 		final double innerDiameter = btn.getInnerDiameter();
 		final double innerRadius = innerDiameter/2;
 	
-		final double sinr = Math.abs(Math.sin(rotation_rad));
-		final double cosr = Math.cos(rotation_rad);
+		final double sinr = Math.abs(Math.sin(angle_offset_rad + view_rotation_rad));
+		final double cosr = Math.cos(angle_offset_rad + view_rotation_rad);
 		final double baseHeightcos = baseHeight*cosr;
 		final double innerHeightcos = innerHeight*cosr;
 		final double flangeHeightcos = flangeHeight*cosr;
 
-		final Coordinate instanceAbsoluteLocation = transformation.transform(Coordinate.ZERO);
+//		System.err.println(String.format("Generating Shapes for RailButtons..."));
+//		System.err.println(String.format("           @  %s", instanceAbsoluteLocation));
+//		System.err.println(String.format("           angle: %f", view_rotation_rad));
```

Comment:
nit: unnecessary debug output left in could be removed

Reply:
oh, good catch.

Reply:
Removed.